README.md

```diff
@@ -6,17 +6,20 @@ Adds a submission terms object to the tender and lot objects, to describe how, w
 
 ## Legal context
 
-In the European Union, this extension's fields correspond to [eForms BG-102 (Submission Terms)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/), although not all the fields have been implemented yet. See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).
+In the European Union, this extension's fields correspond to the following [eForms BG-102 (Submission Terms)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). For correspondences to eForms fields, see [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/latest/en/). For correspondences to Tenders Electronic Daily (TED), see [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/).
```

Comment:
It says "the following" but nothing follows?

Reusing formulation from other extensions.

```suggestion
In the European Union, this extension's fields correspond to [eForms BG-102 (Submission Terms)](https://docs.ted.europa.eu/eforms/latest/reference/business-terms/). See [OCDS for eForms](https://standard.open-contracting.org/profiles/eforms/) for the correspondences to eForms fields. See [OCDS for the European Union](http://standard.open-contracting.org/profiles/eu/master/en/) for the correspondences to Tenders Electronic Daily (TED).
```

Reply:
Ah, just a typo. Fixed in https://github.com/open-contracting-extensions/ocds_submissionTerms_extension/pull/4/commits/51ef97b1d291c7bc99d7eb3852ccdd62ee4d95d0 in order to preserve the formulation that we're now using.